modal/file_io.py

```diff
@@ -374,6 +374,12 @@ def __enter__(self) -> "_FileIO":
     async def __exit__(self, exc_type, exc_value, traceback) -> None:
```

Comment:
Will synchronicity not generate the `__enter__` and `__exit__`?

Reply:
Nice catch, yeah it will